src/frontend/src/lib/components/views/RecoveryPhraseInput.svelte

```diff
@@ -0,0 +1,160 @@
+<script lang="ts">
+  import Tooltip from "$lib/components/ui/Tooltip.svelte";
+  import { InfoIcon } from "@lucide/svelte";
+  import { t } from "$lib//stores/locale.store";
+  import { onMount } from "svelte";
+
+  interface Props {
+    value: string[];
+    showAll?: boolean;
+    disabled?: boolean;
+  }
+
+  let {
+    value = $bindable(),
+    showAll = false,
```

Comment:
Ditto for the other two - show all what? disable what?

Reply:
Maybe in Svelte, it's common to assume that everything inside a component file is an attribute of that component. 

So maybe I should think of these local variables as 

* (RecoveryPhraseInput)_value (makes sense)
* (RecoveryPhraseInput)_showAll (still doesn't make sense)
* (RecoveryPhraseInput)_disabled (makes sense)